### ISSUE TITLE
Incorrect SQL statement using MSSQL

### ISSUE THREAD

**Issue by vindert** [DamienHarper/auditor#34]
| Q                           | A
| ----------------------------| -----------------------
| `auditor-bundle` version    | 4.1.0
| PHP version                 | 8.0.3
| Database                    | MSSQL

#### Summary

Running a migration always terun a ivalid SQL statement for MSSQL.

#### Current behavior

`ALTER TABLE category_audit ALTER COLUMN id INT IDENTITY NOT NULL`
SQL error is:
`Incorrect syntax near the keyword 'IDENTITY`
COLLATE, NOT, NULL, SPARSE or WITH expected, got 'IDENTITY'

#### How to reproduce

running 
`bin/console doctrine:schema:update`
or 
`bin/console make:migration`
when using MSSQL as db

#### Expected behavior

Nothing, when schema is up tot date.

**Comment by vindert** [comment]
@DamienHarper, thanks for all your information.

I've been seeing this issue also on the 3.X branch. When installing the DoctrineAuditBundle I saw the bundle didn't support MSSQL officially. So I didn't bother to report an issue back then.
Now after upgrading to 4.X I didn't find that information anymore. But that was only because of the split package. After now reading the docs thoroughly I found it on the https://damienharper.github.io/auditor-docs/docs/auditor/index.html

The creation of the tables looks ok to me, there is no error upon creating the schema. I also haven't noticed any other bugs during the use of the AuditorBundle

The incorrect SQL statements are generated every time when using 
`bin/console make:migration`
The issue is quickly fixed by the deleting the incorrect statements in the migration.

I understand if you just mark this issue fix for now.  Thanks for searching for all the links.
